src/rgw/rgw_sal_motr.cc

```diff
@@ -399,7 +400,9 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
     MotrAccessKey MGWUserKeys(access_key, secret_key, info.user_id.id);
     store->store_access_key(dpp, y, MGWUserKeys);
   }
-
+ if(!info.user_email.empty()) {
```

Comment:
Style: add space after `if` and before `(`.

Reply:
Fixed